.github/workflows/NodeCI.yml

```diff
@@ -76,7 +76,9 @@ jobs:
         run: pnpm install -D -w eslint@${{ matrix.eslint }}
 
       - name: Test
-        run: pnpm test
+        run: |
+          export NODE_OPTIONS="--max-old-space-size=8192"
```

Comment:
I expect this workaround will no longer be necessary once v5.0.0-165 is released.